### PR TITLE
Viewing and using store credit balance on the storefront

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describe viewing and using the store's credit balance in the storefront from the Review & Payments page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/sales/refunds-customer-account.html

### CODE REVIEW

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for grammar and style. Since guide is for merchants/commerce administrators, storefront customer procedures are third person.

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling, thank you so much for reviewing.

All your suggestions were implemented, please check!

Thank you!

File: src/sales/refunds-customer-account.md [hunk 1]
```diff
@@ -21,11 +21,30 @@ If a refund has been issued for an order, customers can view the refund informat
...
+1. Proceed to checkout.
+
+1. Pass the **Shipping** step.
+
+   If store credit is avalable, you can use it with <span class="btn">Use Store Credit</span>.
+
+   ![]({% link images/images-ee/customer-account-order-refund-from-checkout.png %}){: .zoom}
+   _Store Credit from Review & Payments page_
+
+1. If you want to cancel the use Store Credit payment click <span class="btn">Remove</span> on the **Order Summary** block.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ✅

File: src/sales/refunds-customer-account.md [hunk 2]
```diff
@@ -21,11 +21,30 @@ If a refund has been issued for an order, customers can view the refund informat
...
+
+1. {:.ee-only}You can add the [gift cart]({% link customers/account-dashboard-gift-card.md %}).
+
+**Method 2:** From review & payments page
+
+1. Add product to the cart.
+
+1. Proceed to checkout.
+
+1. Pass the **Shipping** step.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ✅

File: src/sales/refunds-customer-account.md [hunk 3]
```diff
@@ -21,11 +21,30 @@ If a refund has been issued for an order, customers can view the refund informat
...
    ![]({% link images/images-ee/customer-account-store-credit.png %}){: .zoom}
    _Amount refunded to store credit_
+
+1. {:.ee-only}You can add the [gift cart]({% link customers/account-dashboard-gift-card.md %}).
+
+**Method 2:** From review & payments page
+
+1. Add product to the cart.
+
+1. Proceed to checkout.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ✅

File: src/sales/refunds-customer-account.md [hunk 4]
```diff
@@ -21,11 +21,30 @@ If a refund has been issued for an order, customers can view the refund informat
...
    The amount refunded to their store credit appears in the list with the date and time of the action.
 
    ![]({% link images/images-ee/customer-account-store-credit.png %}){: .zoom}
    _Amount refunded to store credit_
+
+1. {:.ee-only}You can add the [gift cart]({% link customers/account-dashboard-gift-card.md %}).
+
+**Method 2:** From review & payments page
+
+1. Add product to the cart.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ✅

File: src/sales/refunds-customer-account.md [hunk 5]
```diff
@@ -21,11 +21,30 @@ If a refund has been issued for an order, customers can view the refund informat
...
 1. If the refund was applied to store credit, chooses **Store Credit** in the left panel.
 
    The amount refunded to their store credit appears in the list with the date and time of the action.
 
    ![]({% link images/images-ee/customer-account-store-credit.png %}){: .zoom}
    _Amount refunded to store credit_
+
+1. {:.ee-only}You can add the [gift cart]({% link customers/account-dashboard-gift-card.md %}).
+
+**Method 2:** From review & payments page
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ✅

File: src/sales/refunds-customer-account.md [hunk 6]
```diff
@@ -21,11 +21,30 @@ If a refund has been issued for an order, customers can view the refund informat
...
+1. From the storefront, log in to account.
 
 1. If the refund was applied to store credit, chooses **Store Credit** in the left panel.
 
    The amount refunded to their store credit appears in the list with the date and time of the action.
 
    ![]({% link images/images-ee/customer-account-store-credit.png %}){: .zoom}
    _Amount refunded to store credit_
+
+1. {:.ee-only}You can add the [gift cart]({% link customers/account-dashboard-gift-card.md %}).
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ✅

File: src/sales/refunds-customer-account.md [hunk 7]
```diff
@@ -21,11 +21,30 @@ If a refund has been issued for an order, customers can view the refund informat
 
 ## View store credit balance and history on the storefront
 
-1. From the storefront, the customer logs in to their account.
+**Method 1:** From customer accoun dashbord
+
+1. From the storefront, log in to account.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ✅

File: src/sales/refunds-customer-account.md [hunk 8]
```diff
@@ -21,11 +21,30 @@ If a refund has been issued for an order, customers can view the refund informat
 
 ## View store credit balance and history on the storefront
 
-1. From the storefront, the customer logs in to their account.
+**Method 1:** From customer accoun dashbord
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ✅

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1190`.

**Pull request merged**